src/storage/src/hummock/file_cache/file.rs

```diff
@@ -82,6 +86,9 @@ impl CacheFile {
     pub async fn open(path: impl AsRef<Path>, options: CacheFileOptions) -> Result<Self> {
         options.assert();
 
+        assert!(options.fs_type != FsType::Tmpfs,
```

Comment:
nits: is `CacheFileOptions::assert` a better place to put this check?

Reply:
Yes, I think that would be a more fitting place.